src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,114 @@
 package core.basesyntax;
 
+import java.util.Arrays;
```

Comment:
According to the checklist, you should not use `Arrays` or any other util class. Please, try to avoid using `Arrays.copyOf()` method in your code and implement it manually instead.